### ISSUE TITLE
Wallace & Gromit: The Curse of the Were-Rabbit Graphical issue

### ISSUE THREAD

**Issue by Triticum0** [xemu-project/xemu#1107]
### Title

https://xemu.app/titles/4b4e0034/#Wallace-Gromit-The-Curse-of-the-Were-Rabbit

### Bug Description

Exploding geometry and weird light source.
![xemu-2022-06-25-23-18-39](https://user-images.githubusercontent.com/71127352/175792137-f6065666-3478-4e9b-a9ad-4cd21d89c4c0.png)
![xemu-2022-06-25-23-18-55](https://user-images.githubusercontent.com/71127352/175792138-b1c2f9df-cd29-4036-8489-47cc18891fde.png)
![xemu-2022-06-25-23-18-59](https://user-images.githubusercontent.com/71127352/175792140-02ae6fdd-43ab-49e4-9f40-f3a29bf42cb0.png)

### Expected Behavior

https://www.youtube.com/watch?v=7YmcXSlY3h4=

### xemu Version

Version: 0.7.49
Branch: master
Commit: https://github.com/mborgerson/xemu/commit/b3abb982e82edbbb142a2c275b893e7f62b10661
Date: Sat Jun 25 06:50:26 UTC 2022

### System Information

Field | Value
-- | --
OS | Windows 10
CPU | AMD Ryzen 5 2600 Six-Core Processo
Graphics Device | NVIDIA GeForce RTX 3060 Ti/PCIe/SSE2
Graphics Driver | 4.0.0 NVIDIA 512.95

### Additional Context

Renderdoc 

Trace

[Wallace & Gromit The Curse of the Were-Rabbit.zip](https://github.com/user-attachments/files/23537137/Wallace.Gromit.The.Curse.of.the.Were-Rabbit.zip)

**Comment by mborgerson** [comment]
Possibly related to #747

**Comment by Triticum0** [comment]
Here a log if it any use
[xemu.log](https://github.com/mborgerson/xemu/files/8994125/xemu.log)

**Comment by Triticum0** [comment]
Retested. still a issue and it is probably related to pr above as changing render fixes the issue temporarily.

**Comment by RinMaru** [comment]
> Retest still a issue and it is probably related to pr above as changing render fixes the issue temporarily.

Testing now but god the FMVs are laggy
Its still an issue on both gl and vk

**Comment by Triticum0** [comment]
Yeah I had leave it for a 30 mins to get ingame once in game fmv can be skipped when starting a new game.

**Comment by RinMaru** [comment]
Yeah I had leave it for a 30 mins to get ingame once in game fmv can be skipped when starting a new game.
Yea it was not fun i should have gotten a soda while i waited. Isnt this on the FMV meta?
and like said earlier switching renderer fixes it for  20 sec then the geometry explodes again

**Comment by Triticum0** [comment]
Adddd

**Comment by RinMaru** [comment]
> Adddd

Just did

**Comment by Triticum0** [comment]
Thanks

**Comment by Itzbewick** [comment]
cut scenes can be laggy make sure to use  resolution scale x2 on display settings  as I found that best for a smooth gameplay hope this helps out
